docs/VideoCapture.md

```diff
@@ -13,9 +13,7 @@
 Then configure your 32blit project with:
 
 ```
-mkdir build
-cd build
-cmake .. -DVIDEO_CAPTURE=true
+cmake . -B build -DVIDEO_CAPTURE=true
```

Comment:
Does this work if we already have a build in `build`? If not how do we fix that?

Reply:
It should work the same as before. Looks like I broke the build with VIDEO_CAPTURE in a previous PR though.

Reply:
The way it worked before is... it didn't. You can't change variables after you've run cmake the first time. I'm not sure why. It just says "variable was not used".

Reply:
Hmm, think it depends on the variable. This one seems to work:
```
~/programs/32blit-beta$ cmake -B build .
-- The C compiler identification is GNU 9.2.1
-- The CXX compiler identification is GNU 9.2.1
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /home/daftfreak/programs/32blit-beta/build
~/programs/32blit-beta$ cmake -B build -DVIDEO_CAPTURE=1 .
-- Configuring done
-- Generating done
-- Build files have been written to: /home/daftfreak/programs/32blit-beta/build
```

I think you get that error from trying to change something like the toolchain file.

Reply:
Hmm... okay then. This is something that I wish was clearer with cmake, but I don't understand the mechanism well enough. I just delete the whole build when I see those errors.

Reply:
I'm in the "delete the whole build" camp here, too. CMake makes that pretty painless to do, since everything is segued into one directory.

Thanks for your feedback here @ali1234 - I'm still learning the ropes with CMake so someone a bit further down that road vetting these changes is much appreciated.

Reply:
You can usually get away with deleting `CMakeCache.txt` and `CMakeFiles/`, but if you're changing something like the toolchain, everything will be getting rebuilt anyway. I'm using the CMake Tools extension in VS Code which handles it automatically.